README.md

```diff
@@ -1,11 +1,97 @@
 
-## Running the Application
+### Prerequisites
+* Java JDK 11+
+* An editor/IDE
+* Gradle
+* Docker
+* OAuth  [(running in a container)](#backing-services)
+* LocalStack [(running in a container)](#backing-services)
+
+### Backing services
+In order to run the service locally, [Nomis OAuth Service](https://github.com/ministryofjustice/nomis-oauth2-server/) and [LocalStack](https://github.com/localstack/localstack) are required. This can be run locally using the provided [docker-compose.yaml](docker-compose.yaml) file which will pull down the latest version.
+
+From the command line run:
+```
+ docker-compose up 
+```
+
+### Build service and run tests
+
+This service is built using Gradle. In order to build the project from the command line and run the tests, you require LocalStack to be running, and use:
+```  
+AWS_PROVIDER=localstack AWS_ACCESS_KEY=local AWS_SECRET_KEY=dev ./gradlew clean build 
+```
+The created JAR file will be named "`offender-assessments-events-<yyyy-mm-dd>.jar`", using the date that the build takes place in the format `yyyy-mm-dd`.
+
+
+### Start the application with H2 database
+
+The configuration can be changed for the api to use an in-memory H2 database by using the spring boot profile `dev`. On the command line run:
+```  
+SPRING_PROFILES_ACTIVE=dev 
+java -jar build/libs/offender-assessments-events-<yyyy-mm-dd>.jar  
+```
+
+### Start the application with H2 database and LocalStack configuration
+
+The application can be configured to use LocalStack and an in-memory H2 DB by using the spring boot profile `local`. Ensure you have LocalStack running, on the command line run:
+```  
+SPRING_PROFILES_ACTIVE=dev AWS_PROVIDER=localstack AWS_ACCESS_KEY=local AWS_SECRET_KEY=dev
+java -jar build/libs/offender-assessments-events-<yyyy-mm-dd>.jar  
+```  
+
+### Additional configuration
+The application is configurable with conventional Spring parameters.  
+The Spring documentation can be found here: https://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
+
+#### Default port
+By default the application starts on port '8080'.   To override, set server.port (e.g. `SERVER_PORT=8099 java -jar build/libs/offender-assessments-api-<yyyy-mm-dd>.jar` )
+
+### Documentation
+The generated documentation for the api can be viewed at http://localhost:8080/swagger-ui.html
+
+### Health
+
+- `/ping`: will respond `pong` to all requests.  This should be used by dependent systems to check connectivity to   
+  offender assessment service, rather than calling the `/health` endpoint.
+- `/health`: provides information about the application health and its dependencies.  This should only be used  
+  by offender assessment service health monitoring (e.g. pager duty) and not other systems who wish to find out the   
+  state of offender assessment service.
+- `/info`: provides information about the version of deployed application.
```

Comment:
I'm not sure if there is an /info endpoint

Reply:
Just had a quick check ,`/info` is returning